pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -65,7 +77,8 @@ def _run_pydoctor(name: str, arguments: Iterable[str], placeholders: Mapping[str
     for argument in arguments:
         args.append(argument.format(**placeholders))
 
-    logger.info(f"Building '{name}' pydoctor API docs as:")
+    build_type = next(iter(arguments))
```

Comment:
I'm not a fan of this:
- this permanently eats the first argument if `arguments` is a generator
- the first argument having any special meaning is not part of this method's documented interface

The first concern can be resolved by changing the type annotation from `Iterable[str]` to `Sequence[str]`, which will disqualify generators. After this change you can just look up `arguments[0]`.

However, as the full list of arguments is logged anyway, do we really need to report the first argument separately?

Reply:
True. That was stupid thing to do :) Thanks!